roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2

```diff
@@ -16,7 +16,7 @@ data:
         <org.csanchez.jenkins.plugins.kubernetes.volumes.PersistentVolumeClaim>
           <mountPath>{{ android_sdk_home }}</mountPath>
           <claimName>{{ ag_android_sdk.pv_claim.name }}</claimName>
-          <readOnly>true</readOnly>
+          <readOnly>{{ ag_android_sdk.pv_claim.read_only }}</readOnly>
```

Comment:
Disabling read-only status of PV invalidates build isolation.
SDK pv is shared between builds. Once writes are enabled, you can inject malicious scripts from your builds or simply install or remove parts of SDK.

Reply:
this was related to jenkins configuration mainly which is not saved after save/apply in jenkins ui

Reply:
But this is not a template for Jenkins. This is a template for android slave configuration.

Reply:
Thats bug. Jenkins configuration could not be saved